dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/ConnectorArguments.java

```diff
@@ -382,6 +387,10 @@ public ConnectorArguments(@Nonnull String... args) throws IOException {
     super(args);
   }
 
+  public void initialize() {
+    searchDefinitions();
+  }
```

Comment:
This exposes additional contract from the object. Should this be done either in the constructor, or in parseArguments()?

Reply:
I need the new contract in order to fail fast if the command-line options are invalid. Doing work in constructor is an anti-pattern, so I would like to avoid this.

As for `parseArguments` - I haven't found such an identifier:
```
$ grep -r parseArguments | wc -l
0
```